gadgets/traceloop/README.mdx

```diff
@@ -146,6 +186,133 @@ Finally, clean the system:
     </TabItem>
 </Tabs>
 
+## Advanced Tracing Scenarios
+
+### Built-in System Call Filtering
+
+For more efficient filtering, traceloop supports built-in syscall filtering at the eBPF level, which reduces overhead and provides cleaner output than post-processing with grep.
+
+<Tabs groupId="env">
+    <TabItem value="kubectl-gadget" label="kubectl gadget">
+        ```bash
+        # Filter for specific syscalls
+        $ kubectl gadget run traceloop:latest -n test-ns --podname my-pod --syscall-filters openat,write,read
+        # Focus on file operations only
+        $ kubectl gadget run traceloop:latest -n test-ns --syscall-filters openat,close,read,write,unlink
+        ```
+    </TabItem>
+
+    <TabItem value="ig" label="ig">
+        ```bash
+        # Filter for specific syscalls (using -c shorthand)
+        $ sudo ig run traceloop:latest -c test-container --syscall-filters openat,write
+        # Focus on file operations only
+        $ sudo ig run traceloop:latest -c test-container --syscall-filters openat,close,read,write,unlink
+        ```
+    </TabItem>
+</Tabs>
+
+**Benefits of built-in filtering:**
+- **Reduced overhead**: Filtering happens at the kernel level, not in userspace
```

Comment:
Another benefit of `--syscall-filters` over `--filter` or `--filter-expr`: it is done before adding the event in the ring buffer, so the ring buffer will have more space for the syscalls that are actually interesting for the user. So we can see a bit further in the past.

Reply:
Okay then